integ-test/src/test/java/org/opensearch/sql/sql/QueryStringIT.java

```diff
@@ -65,4 +66,14 @@ public void wildcard_test() throws IOException {
     JSONObject result3 = executeJdbcRequest(query3);
     assertEquals(10, result3.getInt("total"));
   }
+
+  @Test
+  public void missing_field_test() {
+    String query = StringUtils.format("SELECT * FROM %s WHERE query_string([invalid], 'beer')", TEST_INDEX_BEER);
+    final RuntimeException exception =
+        expectThrows(RuntimeException.class, () -> executeJdbcRequest(query));
+    assertTrue(exception.getMessage()
+        .contains("can't resolve Symbol(namespace=FIELD_NAME, name=invalid) in type env") &&
```

Comment:
It'd be neater, if this was split into separate assertions.

As written, when assertTrue fails it can be for one of two reasons.